packages/algob/src/lib/algo-operator.ts

```diff
@@ -60,4 +70,56 @@ export class AlgoOperatorImpl implements AlgoOperator {
       confirmedRound: txConfirmation[confirmedRound]
     };
   }
+
+  async deployASC (programb64: string, scParams: object, flags: ASCDeploymentFlags
+  ): Promise<ASCInfo> {
+    const program = new Uint8Array(Buffer.from(programb64, "base64"));
+    const encoder = new TextEncoder();
```

Comment:
Create where it's used, not at the top. You don't use it in every line.

Reply:
ok